.github/workflows/rust.yml

```
@@ -0,0 +1,42 @@
+name: Rust
```

```suggestion
name: Publish Rust documentation to GitHub pages
```

And perhaps rename the file as well?